src/store/index.js

```diff
@@ -4,12 +4,15 @@ import migrations from './migrations';
 import modules from './modules';
 import configPersistPlugin from './plugins/persisted-config';
 
-
 Vue.use(Vuex);
 
 const store = new Vuex.Store({
   modules,
-  plugins: [configPersistPlugin('config')],
+  // TODO: https://github.com/brafdlog/budget-tracking/issues/51#issuecomment-658835043
```

Comment:
See comment in this thread: https://github.com/brafdlog/budget-tracking/issues/51#issuecomment-658835043